src/components/ClassSearch.vue

```diff
@@ -16,7 +16,7 @@
           :rows-per-page-items="rowsPerPageItems"
           :pagination.sync="pagination"
           :no-data-text="'No results'"
-          :rows-per-page-text="'Results per page: '"
+          :rows-per-page-text="'Results per page:\xa0'"
```

Comment:
What happened here?

Reply:
Originally, I copy+pasted a non-standard whitespace character. `\xa0` is a non-breaking space, which is a better way of doing this.

Reply:
More context: `eslint` didn't like the string having extra whitespace at the end.